woocommerce/woocommerce
# Add/automated password with password field option

### All Submissions:


### Changes proposed in this Pull Request:

The change is related to this issue: https://github.com/woocommerce/woocommerce/issues/29867

Another option was added to the woocommerce backend to enable an overwrite for the automated account creation on checkout (which is necessary for express checkout like apple pay or google pay).

While an account is automatically created during the express checkout process, the customer will now need to provide an email address and password for any other checkout that needs a customer account.

Closes # .

### How to test the changes in this Pull Request:

1. Go to the woocommerce backend account page
2. activate "auto create account on checkout" & "overwrite automatic account creation"
3. place an item into the cart
4. go to the checkout page
5. you should see the password field for logged out users

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
add new option to allow manual account creation while automatic account creation is active
